### PR TITLE
:sparkles: Add management column for application dependencies table

### PR DESCRIPTION
**Pull request by ibolton336**
- Add management column for application dependencies table
- Add parsed label conversion for dependency labels
https://github.com/konveyor/tackle2-ui/issues/1339

Depends on https://github.com/konveyor/tackle2-addon-analyzer/pull/62

### CODE REVIEW

File: client/src/app/pages/dependencies/dependency-apps-table.tsx [hunk 1]
```diff
@@ -139,6 +140,15 @@ export const DependencyAppsTable: React.FC<IDependencyAppsTableProps> = ({
     },
   } = tableControls;
 
+  const getManagementValue = (appDependency: AnalysisAppDependency) => {
```
Comment (sjd78) [inline client/src/app/pages/dependencies/dependency-apps-table.tsx:143]: What about `s/getManagementValue/isDependencyManaged/` ?

File: client/src/app/pages/dependencies/dependency-apps-table.tsx [hunk 2]
```diff
@@ -139,6 +140,15 @@ export const DependencyAppsTable: React.FC<IDependencyAppsTableProps> = ({
...
+    const hasJavaLabel = appDependency.dependency.labels.some((label) => {
+      const labelValue = getParsedLabel(label).labelValue;
+      return labelValue === "java";
+    });
+    const isJavaFile = dependency.name.endsWith(".jar");
```
Comment (sjd78) [inline client/src/app/pages/dependencies/dependency-apps-table.tsx:148]: 1. Probably better to use `appDependency.dependency.name` just to keep the sourced data in the same place for the function.  Or move the function and the TD render to a column render helper.  Those helpers can be in the same file.  That's what I'm doing in #1540.

2. I don't see any dependency names ending with ".jar".  All the names I see are `mavenGroupId.mavenArtifactId`.  I assume the `appDependency.dependency.provider` string is supposed to be the language, but I've only seen empty string values for my analyses.

**Review by sjd78 — CHANGES_REQUESTED** [review]
Structure looks good.  2 comments.

File: client/src/app/pages/dependencies/dependency-apps-table.tsx (hunk 2 above)
Comment (ibolton336) [reply client/src/app/pages/dependencies/dependency-apps-table.tsx:148]: @sjd78 
@pranavgaikwad mentioned that we only see ".jar" deps when we are using a binary file as the input.

Comment (sjd78) [reply client/src/app/pages/dependencies/dependency-apps-table.tsx:148]: Oh that ***is*** an interesting distinction.  

So only "java" things where sources are not available will have a name with a `.jar` suffix.

**Review by sjd78 — APPROVED** [review]
LGTM

I'm ok with moving along with the current calculation and adjusting with another PR in future if needed.

**Review by sjd78 — APPROVED** [review]
As long as the column `isJavaDependency` tests are good, LGTM

**Pull request merged**